cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaConnectS2IAssemblyOperatorTest.java

```diff
@@ -1073,4 +1099,15 @@ public void testCreateClusterWithSameNameAsConnectFails(VertxTestContext context
             })));
     }
 
+    @Test
+    public void testCreateClusterWithJmxEnabled(VertxTestContext context) {
+        String kcs2iName = "foo";
+        String kcs2iNamespace = "test";
+        KafkaConnectS2I kcs2i = ResourceUtils.createEmptyKafkaConnectS2I(kcs2iNamespace, kcs2iName);
+        kcs2i.getSpec().setJmxOptions(new KafkaJmxOptionsBuilder()
+                .withAuthentication(new KafkaJmxAuthenticationPasswordBuilder().build())
+                .build());
+
+        createCluster(context, kcs2i, false);
+    }
```

Comment:
Could you add this test also for the KafkaConnectAssemblyOperatorTest and for KAfkaMirrorMaker2AssemblyOperatorTest? That would help to have it tested for all of them.

Reply:
yes of course that make sense